.github/workflows/lint-and-tests.yml

```diff
@@ -169,41 +130,16 @@ jobs:
       - name: Run Unit Tests
         # We want to run Unit Tests in every circumstance, including Crowdin PRs and Dependabot PRs to ensure
         # that changes to dependencies or translations don't break the Unit Tests
-        # We want to enforce that the actual `turbo@latest` package is used instead of a possible hijack from the user
-        run: npx --package=turbo@latest -- turbo test:unit -- --ci --coverage
-
-      - name: Start Visual Regression Tests (Chromatic)
-        # This assigns the Environment Deployment for Storybook
-        id: chromatic-deploy
-        # We only need to run Storybook Builds and Storybook Visual Regression Tests within Pull Requests that actually
-        # introduce changes to the Storybook. Hence, we skip running these on Crowdin PRs and Dependabot PRs
-        if: |
-          github.event_name == 'push' ||
-          (github.event_name == 'pull_request_target' &&
-            startsWith(github.event.pull_request.head.ref, 'dependabot/') == false &&
-            github.event.pull_request.head.ref != 'chore/crowdin')
-        # sha reference has no stable git tag reference or URL. see https://github.com/chromaui/chromatic-cli/issues/797
-        uses: chromaui/action@c93e0bc3a63aa176e14a75b61a31847cbfdd341c
+        run: npm run test:ci
+
+      - name: Upload test coverage to Codecov
+        if: ${{ !cancelled() }}
+        uses: codecov/codecov-action@0565863a31f2c772f9f0395002a31e3f06189574 # v5.4.0
         with:
-          workingDir: packages/ui-components
-          buildScriptName: storybook:build
-          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
-          exitOnceUploaded: true
-          onlyChanged: true
-
-      - name: Jest Coverage Comment
-        # We don't need to post the Jest Coverage comment on Crowdin PRs and Dependabot PRs
-        # as in general they introduce no changes to the Unit Tests and the Codebase
-        # We reuse the checks from the Chromatic Deploy step as they're the same conditions
-        if: steps.chromatic-deploy.outcome == 'success'
-        # This comments the current Jest Coverage Report containing JUnit XML reports
-        # and a Code Coverage Summary
-        uses: MishaKav/jest-coverage-comment@d74238813c33e6ea20530ff91b5ea37953d11c91 # v1.0.27
+          files: ./apps/site/lcov.info,./packages/ui-components/lcov.info
+
+      - name: Upload test results to Codecov
+        if: ${{ !cancelled() }}
+        uses: codecov/test-results-action@f2dba722c67b86c6caa034178c6e4d35335f6706 # v1.1.0
```

Comment:
I assume it uses the Codecov App installed on the Org to add the comment to the PR?

Reply:
Depending on what's available, it either
- uses the app installed on the repo
- Comments from the @codecov-commenter account